test/phpunit.xml.dist

```diff
@@ -5,4 +5,9 @@
             <directory>.</directory>
         </testsuite>
     </testsuites>
+    <filter>
```

Comment:
Why is this needed?

Reply:
No need to calculate code coverage for not *.php files (README.md etc)